src/views/open-api-docs-view-v2/utils/parse-open-api-v2-url-context.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiDocsUrlContext } from '../types'
+
+/**
+ * Given an array of URL params, parse the context for an OpenAPI v2 URL, and
+ * Return { isVersionedUrl, versionId, operationSlug } parsed from the URL.
+ */
+export function parseOpenApiV2UrlContext(
```

Comment:
I split this utility function out since the comments felt worth keeping, but also felt like they were cluttering up the `generateStaticPropsVersioned` function body.